### DISCUSSION TITLE
Designing tags with multiple colors to optimize for accessibility (Office hours: Sept 29, 2020)

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#802] [Office hours] [answered]
**The specific questions I have are...**

- Thoughts on annotation tags using multi-colors.
- Goal: optimized for scan-ability to cross reference an annotation and its label. Optimize for accessibility. We only have a limited palette to work with. Eventually, we'll have to cycle colors unless we extend the colors for this tool.
- Want to see what other ideas come to mind.

**Background:** Previously shown this interaction to design systems before. I've scaled back on the use of different colors for annotations based on feedback. And implemented this for feedback from pilot customers.
**Stage of design:** Prototype / iteration
**Links to mockups or prototype:** Current implementation. https://share.goabstract.com/dd65d3af-2bb1-4aa5-a2b3-93836395c301?collectionLayerId=7b25ce4d-8957-415e-8f5f-e98aa6666151&mode=design

Old version that has color labels: https://share.goabstract.com/b64ee4c7-6627-434b-b275-1a530d1bb342?collectionLayerId=3b15c351-ea95-4638-9950-e9d76953c4d2&mode=design

One proposal: https://share.goabstract.com/dd65d3af-2bb1-4aa5-a2b3-93836395c301?collectionLayerId=ab3b0eb1-3056-4eb0-99ef-b8df8610d034&mode=design

I can mock up a new proposal, but wanted to hear any other ideas from the team.

**Comment by kate-porter** [comment]
Hey @elainelylee - thanks for bringing this one back. The design is looking good.

I've left some comments in Abstract, but we'll definitely want to chat more during office hours. A couple of quick questions in the meantime:
- Do you have a sense for the average number of tags a user will have for a given transcript? And is there a maximum?
- Have you done usability testing using a single color or a more limited palette for tags, and if so, was scanability a concern? Any data you have around this will be helpful for us.

**Comment by corinne-nagel** [comment, marked as answer]
Hi @elainelylee can you please share what your final design direction was so everyone can reference? 🙂

**Reply by serifluous** [reply]
Closing due to no response
